### ISSUE TITLE
AcquisitionData storage scheme "memory" leads to Segmentation fault error using TOF-data

### ISSUE THREAD

**Issue by NicoleJurjew** [SyneRBI/SIRF#1060]
After setting the storage scheme for sirf.STIR.AcquisitionData to "memory", when I want to clone a TOF - sinogram, I get the error "Segmentation fault (core dumped)"

```
>>> acq_data = pet.AcquisitionData('data/template_sinogram_create.hs')
>>> acq_ar = acq_data.as_array()
>>> acq_ar.shape
(33, 238, 399, 520)
>>> pet.AcquisitionData.set_storage_scheme('memory')
>>> acq2 = acq_data.clone()
Segmentation fault (core dumped)
```

**Comment by evgueni-ovtchinnikov** [comment]
could be running out of memory case - after increasing my VM memory to 10GB I managed to clone ok

**Comment by KrisThielemans** [comment]
@ALEXJAZZ008008 This might be similar to what you saw

**Comment by ALEXJAZZ008008** [comment]
This is very similar to what I saw, I had to turn memory off. I have enough RAM to hold these objects in memory

**Comment by KrisThielemans** [comment]
Hopefully can be closed since #1208?
